test/kubemark/run-e2e-tests.sh

```diff
@@ -14,16 +14,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-export KUBERNETES_PROVIDER="kubemark"
+# TODO(shyamjvs): This variable should be read from a cloud_provider_config.sh file.
+export CLOUD_PROVIDER="${CLOUD_PROVIDER:-gce}"
+export KUBERNETES_PROVIDER="kubemark/${CLOUD_PROVIDER}"
```

Comment:
Changed it back to the original. And instead incorporated this cloud-provider variance in cluster/kubemark/util.sh .